CONTRIBUTING.md

```diff
@@ -1,34 +1,54 @@
-# Contrubution guide
-This is a contribution guide for this repository. Recommend to read this before working on the project.
+# Contribution guide
+
+ Welcome to Shardeum contribution guide provides the essential information you need to contribute effectively to our repository. Whether you're fixing a bug, adding a new feature, or improving documentation, following these guidelines will streamline the process and ensure smooth collaboration. Please read through this guide carefully before starting your contribution.
 
 # Branch Organization
 
-- This repo has two long running branch, `main` and `dev`. 
-- The two branch are protected.
-- All new branch must be on top of or fork from developement branch `dev`.
-- All branches submiting to development branch must be tested.
-
-This repo try to keep a clean single straight git history line.
-
-# Pull Requests
-
-- Always start PR creation with `Draft` state.
-- Commit should be atomic. 
-- It's okay to commit and push frequeuntly to your branch as long as the branch is rebase and commit are squash before PR.
-- It's okay to have multiple commits if logical. But must keep squashing to single commit as much as possible for PR reviews efficiency. 
-- Recommend to use git commit convention when adding messages to commits but this is not mandatory. [(Read Commit Convention Guide)](https://www.conventionalcommits.org/en/v1.0.0/)
-- Make sure your code comply with formatting/linting configs set by the project.
-- Branches must be tested at least with smoke test mannually.
-- Must include test coverage.
-- Must pass CI/CD pipelines.
-- Reqularly pull updates from develeopment branch `dev` by `--rebase`.
-- Do not merge branches outside of a PR, use rebase.
-- Do not submit to `main`.
-- Undraft and request for review when ready.
-
-# Formatting
-- Uses google typescript prettier settings for the most part.
-- `/*prettier-ignore*/` is often used to bypass really long debug code lines, this is a tip, not an instruction.
-- Do not abuse `/*prettier-ignore*/`.
+
+
+- Maintain two primary branches: `main` for stable releases and `dev` for ongoing development.
+- Both branches are protected to prevent direct commits; all changes must come through pull requests.
+- Always base new branches on the `dev` branch to ensure that features are built on the most recent development work.
+- Ensure that all changes intended for the `dev` branch are thoroughly tested before submission. This includes unit tests, integration tests, and manual checks as necessary.
+
+Ensure this repo is kept clean and linear repository history by using rebase and squash strategies effectively.
+
+
+
+
+
```

Comment:
we can remove the "Branch Organization" section and add just one point to the section "Pull Request Guidelines" saying - "The `dev` branch contains the development version of the code. You can create a branch from `dev` and create a pull request"